app/views/meta/static_files.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Configurable static files.
+
+We have static files and "static" files. The truly static files (e.g., the
+hamburger menu icon or back button) are served directly out of
+resources/static/fixed. The "static" files, configurable by the site admins, are
+served by this view.
+"""
+
+import collections
+
+import django.http
+
+import resources
+import views.base
+
+
+ConfigurableStaticFile = collections.namedtuple(
+    'ConfigurableStaticFile',
+    [
+        'content_type',
+    ])
```

Comment:
is this whitespace correct? it looks weird.

Reply:
I agree it looks weird, but I think that's mainly because it's such a short list (it wasn't really necessary to split this across three lines, but eh, we might add fields later). We do have automated style checking for everything under app/views, and that passed, so I think it's fine.